ietf/templates/500.html

```diff
@@ -1,17 +1,58 @@
-<!DOCTYPE html>
-<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
-<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
-<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
-<!--[if gt IE 8]><!--> <html class="no-js"> <!--<![endif]-->
-    <head>
-        <meta charset="utf-8" />
-        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-        <title>Internal server error</title>
-        <meta name="viewport" content="width=device-width, initial-scale=1" />
-    </head>
-    <body>
-        <h1>Internal server error</h1>
-
-        <h2>Sorry, there seems to be an error. Please try again soon.</h2>
-    </body>
-</html>
+{% extends "base.html" %}
+
+{% block body_class %}template-500{% endblock %}
+
+{% block title %}- 500 - Internal server error{% endblock %}
+
+{% block main_content %}
+<main id="content">
+    <div class="bg-white border-bottom">
+        <div class="container pb-3">
+
+            <nav aria-label="breadcrumbs">
+                <ol class="breadcrumb bg-transparent pt-4 mb-0">
+                    <li class="breadcrumb-item h5">
+                        <a class="text-dark" href="/" aria-label="Home">
+                            <span class="icon ion-android-home" aria-hidden="true"></span>
+                        </a>
+                    </li>
+                </ol>
+            </nav>
+            
+            
+            <div class="u-max-text-width">
+                <h1>HTTP Status Code: 500 Internal Server Error</h1>
+                
+                <p>Ooops, something went wrong. We are looking into it, please try again soon.</p>
+
+                {% if messages %}
```

Comment:
This is not required since we are not going to show any specific error messages to the user.

Reply:
Ignore this. I misunderstood the requirements.